hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/ethereum/EthereumSuite.java

```diff
@@ -227,9 +229,8 @@ final Stream<DynamicTest> baseRelayerCostAsExpected() {
                         .maxPriorityGas(2_000_000L)
                         .gasLimit(1_000_000L)
                         .sending(depositAmount),
-                // The relayer's cost to transmit a simple call with sufficient gas allowance is ≈ $0.0001
-                getAccountInfo(RELAYER)
-                        .has(accountWith().expectedBalanceWithChargedUsd(ONE_HUNDRED_HBARS, 0.0001, 0.5)));
+                // The relayer's cost is no longer applicable for successful transactions.
```

Comment:
Shouldn't these tests here be contingent on the new flag being set?  Which you need to specify explicitly in the test?   And contriwise - shouldn't there still be a test for the new flag _not_ set?

Reply:
Considering that we will be removing the flag when we verify that this is the behaviour we want, I don't see point in investing more time to create more tests here that would be removed in the near future.

Reply:
ok